lib/ansible/modules/source_control/gitlab/gitlab_project_variable.py

```diff
@@ -167,13 +195,27 @@ def native_python_main(this_gitlab, purge, var_list, state):
     existing_variables = [x.get_id() for x in gitlab_keys]
 
     for key in var_list:
+
+        if isinstance(var_list[key], string_types):
```

Comment:
```suggestion
        if isinstance(var_list[key], string_types) or isinstance(var_list[key], integer_types):
```
You also need `from ansible.module_utils.six import integer_types` for that.